### DISCUSSION TITLE
How do I find my Audible Activation code after install is complete

### DISCUSSION THREAD

**Discussion by arunima-92** [audiamus/AaxAudioConverter#68] [Q&A] [answered]
Is there a way to find the Audible Activation code once this application is setup and running. I am looking for it so that I can use ffmpeg to directly convert aax into mp3.

**Comment by audiamus** [comment, marked as answer]
AAX Audio Converter does not install or set up any activation code. That's completely down to Audible. If you use the Audible App, look for `AudibleActivation.sys`
